HW02_A_Graph-Fails.Rmd

```diff
@@ -1,87 +1,110 @@
+
+```
+
+### Graph Fail 2
+Why aren't the points blue? It is making me blue that the points in the graph aren't blue :`(
+
+_Color had to be removed from within aes()._
+
+```{r}
+ggplot(data = mpg) + 
+  geom_point(mapping = aes(x = displ, y = hwy), color = "blue")
+
+```
+
+
+
+### Graph Fail 3
+Two mistakes in this graph. First, I wanted to make the the points slightly bolder, but changing the alpha to 2 does nothing. What does alpha do and what does setting it to 2 do? What could be done instead if I want the points slightly bigger? 
+
+_Alpha sets element transparency; setting it higher (within a range of 0 - 1) increases the opacity of the points._
+_To increase point size, add "size = <size wanted>" within the point geometry function._
+
+Second, I wanted to move the legend on top of the graph since there aren't any points there, putting it at approximately the point/ordered pair (5, 40). How do you actually do this? Also, how do you remove the legend title ("class")? Finally, how would you remove the plot legend completely?
+
+_To move the legend, add the specified points as if the plot x and y axes maximums were 1._
+_To remove the legend title, set legend.position = a blank element._
+_To remove the plot legend completely, set legend.position within theme() equal to "none"._
+
+```{r}
+mpg %>% 
+ggplot() + 
+  geom_point(mapping = aes(x = displ, y = hwy, color = class), alpha = 1, size = 2.5) +
+  theme(legend.direction =  "horizontal") +
+  theme(legend.position = c(.66, .8)) +
+  theme(legend.title = element_blank())
+  
+
+```
+
+### Graph Fail 4
+I wanted just one smoothing line. Just one line, to show the general relationship here. But that's not happening. Instead I'm getting 3 lines, why and fix it please?
+
+_The layer that divided up the colors by drv should have been placed in the geom-point(), because placing it in ggplot places it on a more "upstream" layer upon which geom-smooth() acted._
+
+```{r}
+mpg %>% 
+ggplot(mapping = aes(x = displ, y = hwy)) + 
+  geom_point(aes(color = drv)) + 
+  geom_smooth(se = F) #se = F makes it so it won't show the error in the line of fit
+```
+
+### Graph Fail 5
+I got tired of the points, so I went to boxplots instead. However, I wanted the boxes to be all one color, but setting the color aesthetic just changed the outline? How can I make the box one color, not just the outline? 
+
+_Used "fill" to color the inside of the boxes as well. Removed the legend because it seemed redundant with the x axis._
+
+Also, the x-axis labels were overlaping, so I rotated them. But now they overlap the bottom of the graph. How can I fix this so axis labels aren't on the graph?
+
+_Added a margin with whitespace at the top to avoid overlap._
+
+```{r}
+ggplot(data = mpg, mapping = aes(x = manufacturer, y = cty, color = manufacturer, fill = manufacturer)) + 
+  geom_boxplot() + 
+  theme(legend.position = "none") +
+  theme(axis.text.x = element_text(angle = 45, margin = margin(13, 0, 0, 0, "pt")))
```

Comment:
you could also shift the text with the "hjust=" inside the element.text() function. this is a horizontal justification which just means it aligns it with the edge. the function works on a scale of 0-1.